packages/cli/templates/permission-module/package.json.hbs

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "{{name}}",
+  "description": "The {{id}} module for @backstage/plugin-permission-backend",
+  "version": "{{pluginVersion}}",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "{{license}}",
+{{#if privatePackage}}
+  "private": {{privatePackage}},
+{{/if}}
+  "publishConfig": {
+{{#if npmRegistry}}
+    "registry": "{{npmRegistry}}",
+{{/if}}
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin-module"
```

Comment:
We could populate the `pluginPackage` here as well, to make sure that's correct upfront